.gitignore

```diff
@@ -4,3 +4,6 @@ db.sqlite3
 .cache
 .coverage
 hackman/static/admin
+
+# compiled objects
+*.pyc
```

Comment:
`.pyc` is a python2 thing. `hackman` only runs on python3.

On python3 all `.pyc` files are collected in `__pycache__` which is already ignored.

You can also export the environment variable `PYTHONDONTWRITEBYTECODE` to not write these files in the first place.

In any case this at worst does nothing so it's fine.

Reply:
Ha!  I knew that hackman was python3 only, but (as you can see from the Makefile) my pytest was running the python2 version (until I yelled at it).

I didnt know until I read your message that python3 had changed how they cache the compiled objects..